src/main/java/scm/address/logic/commands/ImportCommand.java

```diff
@@ -104,7 +112,21 @@ public void retrievePersonsFromFile(List<JsonAdaptedPerson> savedPersons)
             }
 
             try {
-                savedPersons.addAll(readPersons(file));
+
+                switch (getFileFormat(file)) {
+                case "json":
```

Comment:
It would be good to have the cases as enums, I think?

Reply:
I'll add Enums as another branch.